src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -278,6 +279,61 @@ protected void printStartupMessage(final ZonedDateTime startDateTime) {
         catch (final Exception e) { /* Unpossible! */ }
     }
 
+    /**
+     * If this tool is either Experimental or Beta, return a warning message advising against use in production
+     * environment.
+     * @param useTerminalColor true if the message should include highlighting terminal colorization
+     * @return a warning message if the tool is Beta or Experimental, otherwise null
+     */
+    protected String getProductionWarning(final boolean useTerminalColor) {
+        final String KNRM = "\u001B[0m"; // reset
+        final String BOLDRED = "\u001B[1m\u001B[31m";
+        final int BORDER_LENGTH = 60;
+
+        String warningMessage = null;
+        if (isBetaFeature()) {
+            warningMessage = String.format(
+                    "\n\n%s   %s\n\n   Warning: %s is a BETA tool and is not yet ready for use in production\n\n   %s%s\n\n",
+                    useTerminalColor ? BOLDRED : "",
+                    Utils.dupChar('!', BORDER_LENGTH),
+                    this.getClass().getSimpleName(),
+                    Utils.dupChar('!', BORDER_LENGTH),
+                    useTerminalColor ? KNRM : ""
+            );
+        }
+        else if (isExperimentalFeature()) {
+            warningMessage = String.format(
+                    "\n\n%s   %s\n\n   Warning: %s is an EXPERIMENTAL tool and should not be used for production\n\n   %s%s\n\n",
+                    useTerminalColor ? BOLDRED : "",
+                    Utils.dupChar('!', BORDER_LENGTH),
+                    this.getClass().getSimpleName(),
+                    Utils.dupChar('!', BORDER_LENGTH),
+                    useTerminalColor ? KNRM : ""
+            );
+        }
+        return warningMessage;
+    }
+
+    /**
+     * If a tool is either Experimental or Beta, log a warning against use in production a environment.
+     */
+    protected void printProductionWarning() {
```

Comment:
Maybe `warnAboutToolStatus()` would be clearer?

Reply:
Done.